index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>my-portfolio-project</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Andika&family=Inter:wght@300;400;500;700&family=Poppins:ital,wght@0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,200&family=Roboto:wght@100;300;400;500;700;900&family=Ubuntu:ital,wght@1,300&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <span>My Logo</span>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+      <div class="nav-list">
+        <ul>
+          <li><a href="#" class="nav-link">Portfolio</a></li>
+          <li><a href="#" class="nav-link">About</a></li>
+          <li><a href="#"  class="nav-link">Contact</a></li>
+        </ul>
+        </div>
+        <div class="hamburger"><i class="fa fa-bars"></i></div>
+      
+    </div>
+  </nav>
+  <!--end of nav section-->
+  <!--start of main section-->
+  <section class="main-wrapper">
+    <div class="main-header">
+      <h1>I'M ABASS <br> Glad to see <br> you!</h1>
+    </div>
+    <section class="content-of-main">
+      <h5>I'm a software developer!<br> I can help you build a product feature or a website. 
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me 
+
+      </h5>
+    </section>
+      <!--social icons -->
+    <section class="social-icons">
+      <div class="icons-header">
+        <h4>LET'S CONNECT</h4>
```

Comment:
## Optional

- [ ] Please change this `<h4>` with `<h2>` because you don't have to skip the headlines level( h1, h2, ... h6). This shows more professionalism.